app/models/manageiq/providers/redfish/inventory/parser/physical_infra_manager.rb

```diff
@@ -48,15 +49,22 @@ def physical_server_details
           :description        => s.Description,
           :location           => format_location(location),
           :location_led_state => s.IndicatorLED,
+          :machine_type       => machine_type(s),
           :manufacturer       => s.Manufacturer,
           :model              => s.Model,
+          :product_name       => s.Name,
           :rack_name          => location.dig("Placement", "Rack"),
           :resource           => server,
+          :room               => location.dig("PostalAddress", "Room"),
           :serial_number      => s.SerialNumber,
         )
       end
     end
 
+    def machine_type(server)
+      server.Processors.Members[0].InstructionSet
```

Comment:
Usually prefer `Members.first.InstructionSet` which also lets you do `Members&.first&.InstructionSet` to handle the case where something is missing